<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatDialog</class>
 <widget class="QDialog" name="chatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame_left">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>191</width>
     <height>601</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="photolabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>121</width>
      <height>561</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_right">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>431</width>
     <height>611</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>421</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendmes">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>540</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upfile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>res/up.ico</normaloff>res/up.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="file">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>res/file.ico</normaloff>res/file.ico</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="filestate">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>文件状态</string>
    </property>
   </widget>
   <widget class="QPushButton" name="audiochoose">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>360</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>res/audio.png</normaloff>res/audio.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="videochoose">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>res/video.ico</normaloff>res/video.ico</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="top_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>631</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
